vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-operators.xml

```diff
@@ -649,12 +649,18 @@
     
     <!-- opext:value($expr as xs:item(), $arg as xs:anyAtomicType?) as item() -->
     <operator name="opext:value">
-        <param name="expr" type="item()"/>
-        <param name="argExpr" type="xs:anyAtomicType"/>
+        <param name="expr" type="json-item()"/>
+        <param name="argExpr" type="item()"/>
```

Comment:
Since for an array the second parameter is an integer and for objects is a string, shouldn't the type be an anyAtomicType instead of an item?